### ISSUE TITLE
Add support for XREAL Adapter updates

### ISSUE THREAD

**Issue by JonasBonno** [MSmithDev/Nreal-Update-Page#1]
Hi,

Thank you for this great project.

Would we be possible to add XREAL Adapter update support as well?

Official update page: hxxps://ota.xreal[.]com/en/adapter-update.html
Firmwares:
- hxxps://media.xreal[.]com/nebula-prod/web/hardware_up/16666964411859360373836744258adapter_11.1.08.001_20221012.bin
- hxxps://media.xreal[.]com/nebula-prod/web/hardware_up/16666964411859360373836744258adapter_11.2.02.001_20230415.bin

**Comment by MSmithDev** [comment]
Yes, this should be fairly easy to add, I will start working on it tomorrow.
Thanks for the feedback!

**Comment by JonasBonno** [comment]
Hi @MSmithDev,

I am looking forward to following you and see if you are able to expand your page with the XREAL Adapter as well.

Hoping for a downgrade method because I am experiencing some sound issues that I believe started for me after updating firmware from 11.1.08.001 to 11.2.02.001.

Let me know if I can help you in any way.
